impl_core/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -119,6 +125,23 @@ private CumulativeMutableViewData(View view, Timestamp start, MetricMap metricMa
       }
     }
 
+    @Override
+    Metric toMetric(Timestamp now, StatsCollectionState state) {
+      if (state == StatsCollectionState.DISABLED) {
+        return null;
+      }
+      MetricDescriptor.Type type = metricDescriptor.getType();
+      List<TimeSeriesCumulative> timeSeriesCumulatives = new ArrayList<TimeSeriesCumulative>();
+      for (Entry<List<TagValue>, MutableAggregation> entry : tagValueAggregationMap.entrySet()) {
+        List<LabelValue> labelValues = MetricUtils.tagValuesToLabelValues(entry.getKey());
+        Point point = MetricUtils.mutableAggregationToPoint(entry.getValue(), now, type);
+        timeSeriesCumulatives.add(
+            TimeSeriesCumulative.create(labelValues, Collections.singletonList(point), start));
```

Comment:
If each `TimeSeries` would always have only one `Point`, why do we need to define `Point`s as a list in the proto?

```proto
// A collection of data points that describes the time-varying values
// of a metric.
message TimeSeries {
  // Must be present for cumulative metrics. The time when the cumulative value
  // was reset to zero. The cumulative value is over the time interval
  // [start_timestamp, timestamp]. If not specified, the backend can use the
  // previous recorded value.
  google.protobuf.Timestamp start_timestamp = 1;

  // The set of label values that uniquely identify this timeseries. Applies to
  // all points. The order of label values must match that of label keys in the
  // metric descriptor.
  repeated LabelValue label_values = 2;

  // The data points of this timeseries. Point.value type MUST match the
  // MetricDescriptor.type.
  repeated Point points = 3;
}
```

Reply:
Multiple reasons why:
1) Same proto can be used for getting data. So in the storage you have more points.
2) If we produce a point each second but export every 10 seconds (not implemented this way right now).

Reply:
SGTM